pkg/agent/cniserver/pod_configuration_linux.go

```diff
@@ -0,0 +1,57 @@
+// +build !windows
+
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cniserver
+
+import (
+	"github.com/containernetworking/cni/pkg/types/current"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/interfacestore"
+)
+
+// connectInterfaceToOVS connects an existing interface to ovs br-int.
+func (pc *podConfigurator) connectInterfaceToOVS(
```

Comment:
Could we move these two funcs to interfaceConfigurator? Originally we hoped to make PodConfigurator OS independent.

Reply:
It's doable. We would need add `ofClient` and `ifaceStore` fields to `ifConfigurator`.

I will try it in next update.

Reply:
@jianjuns ,it might introduce huge changes becasue we will need to move all related functions and fields `interfaceConfigurator`:
- ofClient
- ifaceStore
- disconnectInterfaceFromOVS
- createOVSPort

Do you think we should sill continue the change?

Reply:
I see your points. And seems connectInterfaceToOVS() does not really touch interface/link configuration. Let us keep your current design.